src/UserService.Data.Provider.MsSql.Ef/Migrations/20210316204900_ChangeUserStatus.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace LT.DigitalOffice.UserService.Data.Provider.MsSql.Ef.Migrations
+{
+
+    [DbContext(typeof(UserServiceDbContext))]
+    [Migration("20210316204900_ChangeUserStatus")]
+    public class ChangeUserStatus : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "Status",
+                table: "Users");
+
+            migrationBuilder.AddColumn<int>(
+                name: "Status",
+                table: "Users",
+                nullable: false,
+                defaultValue: 0);
+        }
+    }
```

Comment:
mb you should add Down method?

Reply:
In this case it is possible without a method "down". The migration in this service will be rewritten as in the project service.
If this was the final version of the migration, I would have to add the method "down"

Reply:
Ok, as you say